### PR TITLE
MEN-5062

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

**Review by tranchitella — COMMENTED** [review]
@mzedel I have the (wrong, maybe) impression that this is breaking the file download, which relies on the Cookie and is not in the `/ui` path. What do you think?

**Comment by mzedel** [comment]
@tranchitella thanks for checking that! The e2e tests should catch that if it doesn't work and... for some reason it does work there (a diff from a setup with the `/ui` path limitation [here](https://gitlab.com/Northern.tech/Mender/gui/-/jobs/1629300142/artifacts/file/screenshots/diffs/diff-1632826750654.png)) - however it fails locally for me too 🧐...
Then I'll align the path in useradm and here to be `/`, since we rely on it also in the rest of the communication with the backend...

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -47,7 +47,6 @@ test:lint:
       - screenshots
       - logs
       - junit
-    when: on_failure
```
Comment (tranchitella) [inline .gitlab-ci.yml:50]: Is this a left-over, or intended change?

**Review by tranchitella — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (mzedel) [reply .gitlab-ci.yml:50]: That's intended to allow inspection of screenshots and replays even if a test was just flaky (or in this case passed, but the terminal diff would have been interesting).

**Pull request merged**